build.gradle

```diff
@@ -879,6 +879,7 @@ def getManifestAttributes() {
             "ForceLoadAsMod": !containsMixinsAndOrCoreModOnly.toBoolean()
         ]
     }
+    manifestAttributes += ["Main-class": "kubatech.standalone"]
```

Comment:
Won't this wiped when updating buildscript?

Reply:
It will, but there is no other way I think

Reply:
Well it's your mod, you can maintain it

Reply:
> Well it's your mod, you can maintain it

Ye, its just to show a tea when you double click on a mod in files so not that important anyway